Client/Frontend/Settings/HomepageSettings/HomePageSettingViewController.swift

```diff
@@ -99,8 +99,17 @@ class HomePageSettingViewController: SettingsTableViewController, FeatureFlaggab
         // Setup
         var sectionItems = [Setting]()
 
-        let pocketSetting = BoolSetting(with: .pocket,
-                                        titleText: NSAttributedString(string: .Settings.Homepage.CustomizeFirefoxHome.Pocket))
+        let pocketSponsoredSetting = BoolSetting(with: .sponsoredPocket,
+                                        titleText: NSAttributedString(string: .Settings.Homepage.CustomizeFirefoxHome.SponsoredPocket))
+        pocketSponsoredSetting.enabled = featureFlags.isFeatureEnabled(.pocket, checking: .userOnly)
```

Comment:
I think we should check here for .buildAndUser ? Ex: we might wanna enable sponsored content only on developer and beta build at first?